controller/src/pid.cpp

```diff
@@ -147,9 +117,14 @@ void pid_execute() {
   }
 
   // Update PID Loop
-  sensorValue = Hal.analogRead(DPSENSOR_PIN); // read sensor
+  int16_t sensorValue = Hal.analogRead(DPSENSOR_PIN); // read sensor
   Input = map(sensorValue, 0, 1023, 0, 255);  // map to output scale
   myPID.Compute();                            // computer PID command
   Hal.analogWrite(BLOWERSPD_PIN, static_cast<int>(Output)); // write output
-  send_periodicData(DELAY_100MS, sensorValue, 0, 0);
+
+  // Store sensor readings so they can eventually be sent to the GUI.
+  // TODO(jlebar): Fix pressure/volume/flow data.
```

Comment:
Could you elaborate what the TODO is here? Do we have the data available but just don't put it into the readings, or is this a TODO to determine where to even get the data from?

Reply:
Done.